src/event.cpp

```diff
@@ -293,7 +291,6 @@ void event::per_turn()
    // About once every 10 minutes. Suppress in classic zombie mode.
    if (g->levz >= 0 && one_in(100) && !ACTIVE_WORLD_OPTIONS["CLASSIC_ZOMBIES"]) {
     monster eyebot(GetMType("mon_eyebot"));
-    eyebot.faction_id = faction_id;
```

Comment:
Should probably only have this summon one eyebot: cut down on the swarming folks report.